content/SCALEUIReference/ReportingScreensSCALE.md

```diff
@@ -71,31 +79,77 @@ For example, a system with 18,446,744,073,709,551,615 bytes reports the number a
 
-![SystemReportingProcessUptime](/images/SCALE/23.10/SystemReportingProcessUptime.png "System Reporting")
+{{< trueimage src="/images/SCALE/23.10/SystemReportingProcessUptime.png" alt="System Reporting" id="System Reporting" >}}
 
 {{< /expand >}}
 
 ### ZFS Graphs
 {{< expand "Click Here for More Information" "v" >}}
 ZFS graphs show compressed physical ARC size, hit ratio, demand data, demand metadata, and prefetch data and metadata.
 
-![ZFSReportingARCSizeHit](/images/SCALE/23.10/ZFSReportingActualHitsHitsRate.png "ZFS Reporting ARC Actual Cache Hits Rate and and ARC Hit Rate")
+{{< trueimage src="/images/SCALE/23.10/ZFSReportingActualHitsHitsRate.png" alt="ZFS ARC Actual Cache Hits Rate and and ARC Hit Rate" id="ZFS ARC Actual Cache Hits Rate and and ARC Hit Rate" >}}
+{{< trueimage src="/images/SCALE/23.10/ZFSReportingARCSizeResult.png" alt="ZFS ARC Size and ARC Result" id="ZFS ARC Size and ARC Result" >}}
+
+{{< /expand >}}
+
+## Reporting Exporters Screen
+
+The **Reporting Exporters** screen displays any configured third party reporting exports on the system.
+Exporting enables TrueNAS SCALE to send Netdata reporting metrics to another time-series database.
+For more information, see the Netdata [exporting reference guide](https://learn.netdata.cloud/docs/exporting/exporting-reference).
+
+{{< trueimage src="/images/SCALE/23.10/ReportingExportersScreen.png" alt="Reporting Exporters Screen" id="Reporting Exporters Screen" >}}
+
+Click **Add** to open the **Add Reporting Exporter** screen.
+
+### Add Reporting Exporter
+
+Use the **Add Reporting Exporter** screen to configure third party reporting integrations.
+
+{{< trueimage src="/images/SCALE/23.10/AddReportingExporter.png" alt="Add Reporting Exporter" id="Add Reporting Exporter" >}}
+
+{{< truetable >}}
+| Setting | Description |
+|---------|-------------|
+| **Name** | Enter a unique name for the exporter configuration. If multiple instances are configured, each must have a distinct name. |
+| **Type** | Select the exporter type to configure. The only current supported option is **GRAPHITE**. |
+| **Enable** | Select to send reporting data to the configured exporter. Deselect to disable the explorer without removing configuration. |
+{{< /truetable >}}
+
+Additional settings populate based on the selected **Type** option.
+
+{{< expand "Graphite Settings" "v" >}}
+
+{{< trueimage src="/images/SCALE/23.10/AddReportingExporterGraphite.png" alt="Graphite Settings" id="Graphite Settings" >}}
+
+{{< truetable >}}
+| Setting | Description |
+|-----------|-------------|
+| **Destination Ip** | Required. Enter the IP address of the Graphite server. |
+| **Destination Port** | Required. Enter the port the Graphite server monitors. |
+| **Prefix** | Enter the prefix to add to all Netdata metrics sent to Graphite. Defaults to *dragonfish*. |
+| **Hostname** | Enter the hostname to add to all metrics, after the prefix, for sending data to the Graphite server. Defaults to *truenas* |
```

Comment:
This sentence is awkward, like you had a thought, then got distracted. Not sure how to parse it.
```suggestion
| **Hostname** | Enter the host name to add to all metrics, after the prefix, for sending data to the Graphite server. Defaults to *truenas* |
```